views/default/input/form.php

```diff
@@ -23,17 +25,40 @@
 $vars = array_merge($defaults, $vars);
 
 $vars['class'] = elgg_extract_class($vars, 'elgg-form');
-$vars['action'] = elgg_normalize_url($vars['action']);
-$vars['method'] = strtolower($vars['method']);
 
-$body = $vars['body'];
+$vars['action'] = elgg_normalize_url(elgg_extract('action', $vars, ''));
+unset($vars['action_name']);
```

Comment:
why unsetting the action_name here? it is not used before... shouldn't it be removed at all?